engine/apps/slack/alert_group_slack_service.py

```diff
@@ -0,0 +1,136 @@
+import logging
+
+from django.apps import apps
+
+from apps.slack.constants import SLACK_RATE_LIMIT_DELAY
+from apps.slack.slack_client import SlackClientWithErrorHandling
+from apps.slack.slack_client.exceptions import (
+    SlackAPIChannelArchivedException,
+    SlackAPIException,
+    SlackAPIRateLimitException,
+    SlackAPITokenException,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AlertGroupSlackService:
+    def __init__(self, slack_team_identity, slack_client=None) -> None:
+        self.slack_team_identuty = slack_team_identity
```

Comment:
```suggestion
        self.slack_team_identity = slack_team_identity
```

Reply:
actually, do we need to set `self.slack_team_identity` here? I don't see `slack_team_identuty` referenced anywhere else in this PR

Reply:
bump